packages/kit/src/runtime/client/client.js

```diff
@@ -613,10 +613,10 @@ export function create_client({ target, session, base, trailing_slash }) {
 
 			if (import.meta.env.DEV) {
 				try {
-					fetcher.increment();
+					increment();
```

Comment:
Following previous comment:
```suggestion
					lock_fetch();
```